app/js/containers/Seller/4_SellerMarginContainer.jsx

```diff
@@ -26,6 +30,11 @@ class SellerMarginContainer extends Component {
     } else {
       this.setState({ready: true});
     }
+
+    this.props.fetchPrices({
+      from: [this._getSymbol()],
```

Comment:
The problem with this solution is that the price changes all the time and if you left the page open for 1 hours the data displayed might not be accurate.

What about when we do the initialisation we also start a scheduler who is going to fetch prices for all token/currency periodically => e.g every 1 or n minutes.

We can then attach the prices to the different tokens we have

What do you think?

Reply:
If we do that, don't forget to kill the interval on component unmount

Reply:
I would not even do that :)
I would imagine the token always have a set of price attach to them that is up to date

So not related to the specific component but to the App

Reply:
Ah ok, yeah, if the fetch is in the App, it's ok then